server/main.lua

```diff
@@ -1697,22 +1697,42 @@ end)
 QBCore.Functions.CreateCallback('getWeaponInfo', function(source, cb)
     local Player = QBCore.Functions.GetPlayer(source)
     local weaponInfos = {}
-    for _, item in pairs(Player.PlayerData.items) do
-        if item.type == "weapon" then
-            local invImage = ("https://cfx-nui-%s/html/images/%s"):format(Config.InventoryForWeaponsImages, item.image)
-            if invImage then
-                local weaponInfo = {
-                    serialnumber = item.info.serie,
-                    owner = Player.PlayerData.charinfo.firstname .. " " .. Player.PlayerData.charinfo.lastname,
-                    weaponmodel = QBCore.Shared.Items[item.name].label,
-                    weaponurl = invImage,
-                    notes = "Self Registered",
-                    weapClass = "Class 1",
-                }
-                table.insert(weaponInfos, weaponInfo)
-            end
-        end
-    end
+	if not Config.InventoryForWeaponsImages == "ox_inventory" then

```

Comment:
Generally, when there is something you want to be an option to support, don't ask for "not". I'd suggest to change this around to be that if the config is set to ox_inventory, then use the ox stuff. Otherwise, use the default stuff.